docs/quickstart_databricks.md

```diff
@@ -1,7 +1,6 @@
 ---
 layout: default
 title: Quick Start Guide With Databricks
-nav_order: 3
 ---
 
 # Feathr Quickstart Guide for Databricks
```

Comment:
Shall we align the title? Feathr Quickstart Guide With Databricks

Reply:
solved